Library/Homebrew/download_strategy.rb

```diff
@@ -665,6 +665,11 @@ def initialize(name, resource)
     @url = @url.sub("svn+http://", "")
   end
 
+  def cache_filename
+    # "@" is a special character for Subversion. Keep it out of the cache dir name.
```

Comment:
Feels like this comment should be for safe download name

Reply:
Moved comment to `safe_download_name`.